### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [MichaelCurrin/dev-cheatsheets#45] [Q&A] [answered]
Hi everyone.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by MichaelCurrin** [comment, marked as answer]
Hi thanks for the interest. Yes it is and I am still adding content to it.

If you want to pick up and issue, I have one annoying one that needs attention https://github.com/MichaelCurrin/dev-cheatsheets/issues/41

**Reply by ghost** [reply]
Hi MichaelCurrin.

> Hi thanks for the interest. Yes it is and I am still adding content to it.
> If you want to pick up and issue, I have one annoying one that needs attention https://github.com/MichaelCurrin/dev-cheatsheets/issues/41

thank you for feedback.

**Comment by MichaelCurrin** [comment]
Or this one to add quality check https://github.com/MichaelCurrin/dev-cheatsheets/issues/29
